packages/build/src/telemetry/plugins.js

```diff
@@ -54,6 +63,15 @@ const netlifyTelemetry = {
   },
 }
 
+function osType(platform) {
+  if (platform == 'darwin') {
+    return 'MacOS'
+  } else if (platform == 'win32' || platform == 'win64') {
+    return 'Windows'
+  }
+  return 'Linux'
```

Comment:
It would be interesting to log other OS as well, although much less likely, such as BSD.
https://nodejs.org/api/os.html#os_os_platform

Also please note `win64` does not exist. `os.platform()` is always `win32` regardless of the CPU architecture.